### PR TITLE
Update plotting

### LINKED ISSUE #63: Fix ugly seaborn/pyplot code for LogQuestion
The main problem is that we're converting to a log scale ourselves, rather than having seaborn/pyplot do it (note that if you just use `xscale("log")`, seaborn/pyplot are really bad at making a pretty graph)

We should either do this more elegantly or figure out how to have seaborn/pyplot do it for us

### LINKED ISSUE #64: Make graphing API consistent between ContinuousQuestion and LogQuestion
there are currently some kludgy extra parameters for ContinuousQuestion

### LINKED ISSUE #69: Use plotnine instead of seaborn/pyplot

### LINKED ISSUE #74: Clean up + add lockdown series notebook

### LINKED ISSUE #112: Merge the two community predictions notebooks

### LINKED ISSUE #139: Simple, consistent API for graphing predictions on continuous questions
Right now graphing predictions on continuous questions is really confusing.

I mostly created the API, and [here](https://colab.research.google.com/drive/17w7YUJJA3gJD8ketp6ZQtDiaeV1foTc7#scrollTo=UBjzm9nhdyD2&line=1&uniqifier=1) I couldn't remember how to get my prediction and the community prediction on the same graph :|

I think we should probably just have one function that users are meant to use, `show_prediction(self, samples=None, show_community=True)`

This new function would be largely the same as the current `show_submission` as seen in the last 2 plots here: https://github.com/oughtinc/ergo/blob/master/notebooks/community-distributions.ipynb

i.e. you should be able to plot the samples, the logistic mixture, and the community prediction together

This would replace:
1. `show_performance`: we don't really use this right now and should just get rid of it I think
2. `show_submission`: this is currently what we mainly use. But I think `show_prediction` is a better name than `show_submission`
3. `show_community_prediction`: currently how you see the community prediction by itself. I think just calling `show_prediction()` or `show_prediction( show_community=True)` would simplify things
4. the current `show_prediction`. This isn't actually meant to be used by users currently (it shows a prediction based on mixture params, we just use it as part of other functions)

Maybe most important, our new API should work consistently across all 3 types of continuous questions; there's a lot of heterogeneity right now

### LINKED ISSUE #140: Make graphs zoom in on the data better, or allow users to adjust zoom
Sometimes they're pretty absurdly zoomed out, as [here](https://colab.research.google.com/drive/17w7YUJJA3gJD8ketp6ZQtDiaeV1foTc7#scrollTo=tO03It_Le3ZT&line=14&uniqifier=1).

Marking as `el-paso-urgent` because I found this annoying on an El Paso question; assigning to Dan because it's a plotting thing and may be solved by ggplot

### PR DESCRIPTION
**Pull request by djthorne**
This is the first step towards coherent plotting across questions. Will close issues in subsequent PR when I document better

Fixes #139 
Fixes #140 
Fixes #74
Fixes #69 
Fixes #64 
Fixes #112 
Fixes #63

### CODE REVIEW

**Comment by brachbach** [comment]
@stuhlmueller asked me to try out this PR for plotting a prediction I was working on. I noticed some issues while doing so:

1. First I tried just using this PR to make plots. This didn't work: Screen Shot 2020-04-21 at 1 52 54 PM

[notebook](https://colab.research.google.com/drive/1tn7LUibCh881BhyzFlcCPXe_ZFwylAsz#scrollTo=a_2oH2vfhzek)
2. Then I tried running the TestVisualPandemic tests. They all failed, some for reasons that are not entirely trivial. I made [this branch](https://github.com/oughtinc/ergo/tree/update_plotting_visual_tests) to try to get them to work, but I couldn't quickly get them to work

@djthorne I assume you'll take it from here

File: ergo/__init__.py [hunk 1]
```diff
@@ -31,3 +33,5 @@
     to_float,
     uniform,
 )
+
+warnings.filterwarnings(module="plotnine", action="ignore")
```
[the remark this answers is not in the document]
Comment (djthorne) [reply ergo/__init__.py:37]: I think your right, perhaps we will just add that in sample notebooks to be more explicit

File: ergo/metaculus.py [hunk 2]
```diff
@@ -367,8 +373,8 @@ def sample_normalized_community(self):
         # how spread out we actually expect the probability mass to be
         outside_range_scale = 0.02
 
-        sample_below_range = -abs(np.random.logistic(0, outside_range_scale))
-        sample_above_range = abs(np.random.logistic(1, outside_range_scale))
+        sample_below_range = -abs(np.random.logistic(0, outside_range_scale))  # type: ignore
```
[the remark this answers is not in the document]
Comment (djthorne) [reply ergo/metaculus.py:409]: I can't claim that section. Perhaps @brachbach has an opinion?

**Comment by brachbach** [comment]
@djthorne I fixed 3. from above, the testing issue, in https://github.com/oughtinc/ergo/pull/126

(I had written the test incorrectly initially)

File: ergo/metaculus.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (brachbach) [reply ergo/metaculus.py:418]: Honestly I don't remember. I agree that I should have left a comment, but no longer remember what it should say

**Pull request merged**